dev-util/kdstatemachineeditor/files/fix_hardcoded_installation_dirs-1.2.1.patch

```diff
@@ -0,0 +1,29 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -10,6 +10,9 @@ if(POLICY CMP0063)
+   cmake_policy(SET CMP0063 NEW)
+ endif()
+ 
++#Let's deal with upstream's harcoded dirs.
++include(GNUInstallDirs)
++
+ # Version setup
+ set(KDSME_VERSION_MAJOR "1")
+ set(KDSME_VERSION_MINOR "2")
+@@ -92,11 +95,11 @@ set(CMAKE_INCLUDE_CURRENT_DIR ON)
+ set(CMAKE_INCLUDE_CURRENT_DIR_IN_INTERFACE ON)
+ set(CMAKE_AUTOMOC TRUE)
+ 
+-set(BIN_INSTALL_DIR bin)
+-set(LIB_INSTALL_DIR lib CACHE STRING "Library install destination.")
+-set(INCLUDE_INSTALL_DIR include/${CMAKE_PROJECT_NAME})
+-set(XDG_DATA_INSTALL_DIR share/${CMAKE_PROJECT_NAME})
+-set(QCH_INSTALL_DIR "doc" CACHE STRING "Install location of Qt Assistant help files.")
++set(BIN_INSTALL_DIR ${CMAKE_INSTALL_FULL_BINDIR})
```

Comment:
The right approach for an upstream patch rather than duplicating its value would be simply replacing every instance of `BIN_INSTALL_DIR` with `CMAKE_INSTALL_BINDIR` (not FULL!).